data-collector.go

```diff
@@ -89,7 +89,9 @@ func (dcc *DataCollectorClient) Update(nodeName string, body interface{}) (*http
 		defer res.Body.Close()
 		statusCode = res.StatusCode
 	}
-	requests <- &request{Method: req.Method, Url: req.URL.String(), StatusCode: statusCode}
+	if !config.RandomData {
```

Comment:
What do we gain by not sending this to the `requests` channel when generating random data? The `requests` channel is only used to keep a running count of each request type.